### PR TITLE
fix publish okteto release for windows

### PR DESCRIPTION
**Pull request by AdrianPedriza**
# Context
new okteto release binaries related to windows are not available since okteto CLI `2.3.3`, instead of that these binaries are under the channel `stable` using `https://downloads.okteto.com/cli/stable/okteto.exe`.

# Proposed changes
add built artifacts into the proper bucket folder in order to make public stable releases on windows using `https://downloads.okteto.com/cli/okteto.exe` url

Fixes #https://github.com/okteto/app/issues/6414

### CODE REVIEW

**Review by jLopezbarb — APPROVED** [review]

File: scripts/ci/release.sh [hunk 1]
```diff
@@ -180,13 +185,19 @@
...
                 if [ "$tag" = "$latest" ]; then
-                        gsutil -m rsync "gs://$BIN_BUCKET_NAME" "gs://$BIN_BUCKET_ROOT"
+                        gsutil -m rsync "gs://$BIN_BUCKET_NAME" "gs://$BIN_BUCKET_ROOT_WITH_CHAN"
                 fi
 
                 gsutil -m -h "Cache-Control: no-store" -h "Content-Type: text/plain" cp "${version_file}" "gs://${VERSIONS_BUCKET_FILENAME}"
                 echo "${chan} channel updated with ${tag}"
         done
 
+        if [ "$is_oficial_release" = true ] ; then
```
[the remark this answers is not in the document]
Comment (AdrianPedriza) [reply scripts/ci/release.sh:195]: I think that block is executed also for betas (see workflow for `2.17.0-beta.1` https://app.circleci.com/pipelines/github/okteto/okteto/10963/workflows/4bfec636-5198-4346-a19a-836f02a75677/jobs/34538) if so, we would be uploading beta artifacts into root I think and using it as official releases in `https://downloads.okteto.com/cli/stable/okteto.exe`. Dont we?

[the remark this answers is not in the document]
Comment (AdrianPedriza) [reply scripts/ci/release.sh:195]: ah okay, you mean inside that block and also checks that `is_oficial_release = true`?

[the remark this answers is not in the document]
Comment (maroshii) [reply scripts/ci/release.sh:195]: Yeah, We should be able to push to an older stable version without updating latest. FWIW, I think we should get rid of the mutable root binaries and just stick with version paths but it's a breaking change

[the remark this answers is not in the document]
Comment (AdrianPedriza) [reply scripts/ci/release.sh]: 8b829f24d0fb54d239bed9ecd43fb233f1c835d4 . About modifying our current storage logic we could make a proposal adding pros and cons of it. If not we can always address this issue in the future

**Review by maroshii — APPROVED** [review]

**Pull request merged**